helix-term/src/commands.rs

```diff
@@ -2225,10 +2229,10 @@ fn buffer_picker(cx: &mut Context) {
 
             let mut flags = Vec::new();
             if self.is_modified {
-                flags.push("+");
+                flags.push(self.modified_indicator.to_string());
             }
             if self.is_current {
-                flags.push("*");
+                flags.push(self.current_indicator.to_string());
```

Comment:
If we do this then we have to modify the `join` on line 2241 so that it gets a `Vec<String>` instead of a `Vec<&String>`. Happy to make this change since I'm not sure which approach is more idiomatic. (Also, do you recommend an `into()` for this or a `map()`?)